test/pr/expected/disable-ops-agent.txt

```diff
@@ -0,0 +1,637 @@
+module.startup.data.google_storage_bucket.existing_bucket[0]: Reading...
+module.aiinfra-network.module.default_vpc[0].data.google_compute_subnetwork.primary_subnetwork: Reading...
+module.aiinfra-network.module.default_vpc[0].data.google_compute_network.vpc: Reading...
+module.startup.data.google_storage_bucket.existing_bucket[0]: Read complete after 1s [id=aiinfra-terraform-gce-ai-infra]
+module.aiinfra-network.module.default_vpc[0].data.google_compute_network.vpc: Read complete after 1s [id=projects/gce-ai-infra/global/networks/default]
+module.aiinfra-network.module.default_vpc[0].data.google_compute_subnetwork.primary_subnetwork: Read complete after 1s [id=projects/gce-ai-infra/regions/us-central1/subnetworks/default]
+module.aiinfra-mig.data.google_compute_image.compute_image: Reading...
+module.aiinfra-mig.data.google_compute_image.compute_image: Read complete after 0s [id=projects/ml-images/global/images/c2-deeplearning-pytorch-1-12-cu113-v20230126-debian-10]
+
+Terraform used the selected providers to generate the following execution
+plan. Resource actions are indicated with the following symbols:
+  + create
```

Comment:
Looks like you are using this as an expected output to match 1 to 1. But as we add more functionality, the output would change and would break this test. 
Is there any other way we can validate this test. Like instead of full 1 on 1 match, we can do string contains or something of that short ?

Reply:
yeah that got real ugly real quick didn't it. I changed it so that the test checks whether an expected json file is a subset of the tfplan output (now also a json file). this way, tests will only fail if those specific values are change/removed and not if new values/modules are added